symplyphysics/laws/hydro/hydrostatic_pressure_from_density_and_depth_acceleration.py

```diff
@@ -0,0 +1,28 @@
+from sympy import Eq, solve
+from symplyphysics import units, Quantity, Symbol, print_expression, validate_input, validate_output
+
+# Description
+# Law: P = ρ * g * h
```

Comment:
Derive existing law from this.

Reply:
> Derive existing law from this.

Get the law in the file "hydrostatic_pressure_from_density_and_depth" from the file "hydrostatic_pressure_from_density_and_depth_acceleration"?

Reply:
Yes.